pkg/utils/transform/transform.go

```diff
@@ -65,11 +65,44 @@ func InjectNamespaceConditional(preserveNamespace, targetNamespace string) mf.Tr
 		if ok && val == "true" {
 			return nil
 		}
-
 		return tf(u)
 	}
 }
 
+func InjectNamespaceRoleBinding(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		switch strings.ToLower(u.GetKind()) {
+		case "rolebinding":
```

Comment:
shouldn't we handle clusterrolebindings as well?

Reply:
@sthaha `ClusterRoleBindings` subjects are handled by manifestival, but it doesn't care for `rolebindings` 
they should have their reasons